### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by CarlesAB**

### CODE REVIEW

File: pandlas/SqlRace.py [hunk 1]
```diff
@@ -325,7 +329,7 @@ def __init__(
         if self.mode == "r":
             self.load_session(session_key)
         elif self.mode == "w":
-            self.create_sqlrace()
+            self.create_sqlrace(ip_address)
```
Comment (owentmfoo) [inline pandlas/SqlRace.py:332]: not needed as self.ip_address is set in L321?

File: pandlas/SqlRace.py [hunk 2]
```diff
@@ -367,7 +371,7 @@ def start_recorder(self, port=7300):
             port += 1
         logger.info("Opening server lister on port %d.", port)
```
Comment (owentmfoo) [inline pandlas/SqlRace.py:372]: update log message to include ip address as well

**Review by owentmfoo — CHANGES_REQUESTED** [review]

File: pandlas/SqlRace.py (hunk 1 above)
Comment (CarlesAB) [reply pandlas/SqlRace.py:332]: removed ip_address from the self.create_sqlrace()

File: pandlas/SqlRace.py (hunk 2 above)
Comment (CarlesAB) [reply pandlas/SqlRace.py:372]: logger.info("Server Listener IPAddress %s.", self.ip_address)

Added.

**Review by owentmfoo — APPROVED** [review]

**Pull request merged**